modules/media/jsx/mediaIndex.js

```diff
@@ -0,0 +1,32 @@
+/* global DynamicDataTable MediaUploadForm formatColumn */
+
+$(function() {
+  loris.hiddenHeaders = ['Cand ID', 'Session ID'];
```

Comment:
Agreed, but these fields were just moved and I think changing them is outside the scope of this PR. It would require more changes in the media module. IMO it should be done in another PR.